_projects/2018/heliopy/heliopy-units.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Using physical units with HelioPy
+desc: Convert HelioPy's data import methods to return data that has units attached.
+requirements:
+ - Some experience using python
+difficulty: low
+issues:
+ - https://github.com/heliopython/heliopy/issues/291
+mentors:
+ - dstansby
```

Comment:
As we "require" two mentors per project I'm happy to co-mentor this one unless you find a person who could commit more time.

Reply:
Thanks - I will put your name down for now. I was thinking of reaching out to sunpy/astropy/others dev lists to see if anyone else would be interested in co-mentoring, but haven't got round to it yet.